_posts/2018/2018-12-21-how-sdslabs-works.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: How SDSLabs works - 2018
+excerpt: "An insight into the tools used inside SDSLabs"
+---
+
+SDSLabs is roughly run as two cells (programming/design) that collaborate with each other on everything. This blog post will take a look at the various tools, technologies, and applications that we use on a daily basis. It is our hope that some other groups in the campus might be interested in this. We’re open to the inquiry about any of this, and you can reach us anytime at [chat.sdslabs.co](https://chat.sdslabs.co/) or [contact@sdslabs.co.in](mailto:contact@sdslabs.co.in).
+
+**Mailing List**: Our mailing list, like most other groups', runs on Google Groups. This is our primary means of communication. We send out a lot of mails, and email-overload is an actual problem for people who just join. All notifications regarding discussion of projects or general meetings are sent over the mailing list. We didn't want to disturb our alumni over our regular meetings, so we now have a separate groups for current members and the alumni.
```

Comment:
Is this still valid? I don't see much activity, maybe Slack should be higher?

Reply:
Yeah, adding overload messages to slack.